gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/MapContainerFragment.java

```diff
@@ -77,6 +79,28 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
     mapContainerViewModel = getViewModel(MapContainerViewModel.class);
     homeScreenViewModel = getViewModel(HomeScreenViewModel.class);
     mainViewModel = getViewModel(MainViewModel.class);
+    Single<MapAdapter> mapAdapter = mapProvider.getMapAdapter();
```

Comment:
Here, I've just moved the subscriptions into the onCreate method. There's no meaningful way to port them into the MapAdapter, that I know of, since it's injected. I'm not certain that this makes much of a difference from the previous implementation, since onMapReady is (hopefully?) only called once per Fragment instantiation anyway (?) but this should help make certain we're only subscribing once in the lifecycle, or that's the hope.